internal/rundata/dut.go

```diff
@@ -39,8 +39,8 @@ type dutInfo struct {
 
 // setFromComponentChassis sets dutInfo from the first component of type CHASSIS.
 //
-//   - vendor from mfg-name (Arista, Cisco).
-//   - model from either description (Cisco, Juniper) or part-no (Arista).
+//   - vendor from mfg-name (Arista, Cisco, newer Nokia).
```

Comment:
nit: remove "newer"? so that we don't need to remember to update it later (next build will support it).